cadence/errors.py

```diff
@@ -106,6 +106,12 @@ class ClientVersionNotSupportedError(Exception):
     supported_versions: str
 
 
+@dataclass
+class RPCError(Exception):
+    message: str
+    code: grpc.StatusCode
```

Comment:
I think the idea of this file is to define types independent of thrift/grpc. So it's better to not import any grpc types.

Reply:
Makes sense. I'll start figuring out how to do the mapping